docs/start/authentication/graphql-storefront.mdx

```diff
@@ -66,17 +66,226 @@ content-type: application/json
+{
+    "data": {
+        "loginWithCustomerLoginJwt": {
+            "customer": {
+                "entityId": 22,
+                "email": "user@email.com"
+            },
+            "customerAccessToken": {
+                "value": "ee973d6566a7c691af4602f344d866ecaa45bdcbdf2075d29d5e70fd0716b7c5_1721657159",
+                "expiresAt": "2024-07-22T14:05:59Z"
+            }
+        }
+    }
+}
+```
+</Tab>
+</Tabs>
+
+
+##### Using a Customer Access Token
+
+You can assign the customer access token obtained using one of the mutations described above to the `X-Bc-Customer-Access-Token` header for future queries. You can use this token for a specific shopper session access until it is invalidated. The token is invalidated when you use the logout mutation with the `X-Bc-Customer-Access-Token` header.
+
+<Tabs items={['Request', 'Response']}>
+<Tab>
+
+```graphql filename="Example customer query" showLineNumbers copy
+POST https://{{storeDomain}}/graphql
+Authorization: Bearer {Storefront token}
+X-Bc-Customer-Access-Token: {Customer access token}
+accept: application/json
+content-type: application/json
+
+query CustomerAttributes {
+  customer {
+    firstName
+    lastName
+    email
+    entityId
+    customerGroupId
+    attributeCount
+    attributes {
+      shirtSize: attribute(entityId:123) {
+        entityId
+        value
+      }
+      favoriteColor: attribute(entityId:456) {
+        entityId
+        value
+      }
+    }
+  }
+}
+```
+</Tab>
+<Tab>
+
+```json filename="Example customer query" showLineNumbers copy
+{
+    "data": {
+        "customer": {
+            "firstName": "string",
+            "lastName": "string",
+            "email": "user@email.com",
+            "entityId": 21,
+            "customerGroupId": 2,
+            "attributeCount": 0,
+            "attributes": {
+                "shirtSize": {
+                    "entityId": 123,
+                    "value": medium
+                },
+                "favoriteColor": {
+                    "entityId": 456,
+                    "value": blue
+                }
+            }
+        }
+    }
+}
+```
+</Tab>
+</Tabs>
+
+
 ### Using auto-generated tokens on Stencil storefronts
 
 On Stencil storefronts, you can access a token at render time and pass the token to client-side code using the `{{settings.storefront_api.token}}` Handlebars property. This auto-generated token has an expiry period of 24-48 hours and will rotate before expiration.
 
+
 ## Customer impersonation tokens
 
-You can use customer impersonation tokens to authenticate requests to the GraphQL Storefront API in server-to-server and headless interactions. The API will reject any customer impersonation token-authenticated requests that originate from a web browser and are not proxied. Use these tokens in NextJS applications; NextJS proxies requests by default.
+<Callout type="warning">
```

Comment:
This section should have a callout that not all customer features will be supported when using a CIT in the future; features that are more security-sensitive will only work with CAT.